### ISSUE TITLE
LoadingIcon and register login page implementations

### ISSUE THREAD

**Issue by Mert18** [hire-a-senior/frontend#22]

**Comment by ManishMadan2882** [comment]
I can work on this, please assign

**Comment by Mert18** [comment]
Of course, since I am still working on register and login pages, you do not need to add it there. Just create a LoadingIcon component and it will be fine.

**Comment by Mert18** [comment]
It is desirable that loading icon includes purple colors defined in "tailwind.config.ts"
